pages/getting-started/install-memgraph/docker.mdx

```diff
@@ -404,7 +356,7 @@ Linux](https://docs.microsoft.com/en-gb/windows/wsl/install-win10#step-4---downl
 and enable experimental features in Docker Desktop, under *Settings* -> *Docker
 Engine*, change *experimental* to *true*.
 
-### Issues when connecting to Memgraph
+### Issues when connecting to Memgraph Platform
```

Comment:
This section should be aligned with the new Memgraph Platform, meaning this kind of section can be created in the Docker Compose for Memgraph platform (similar things might happen - e.g., https://memgraph.slack.com/archives/C066Q419WDC/p1706027145425559?thread_ts=1705943595.715429&cid=C066Q419WDC). Here, I would then again set title to 'up to version 2.14'

Reply:
Done.

Reply:
Done.